Client/Frontend/Settings/SettingsTableViewController.swift

```diff
@@ -699,11 +699,14 @@ class SettingsTableSectionHeaderFooterView: UITableViewHeaderFooterView {
 
     override init(reuseIdentifier: String?) {
         super.init(reuseIdentifier: reuseIdentifier)
-        contentView.backgroundColor = SettingsUX.TableViewHeaderBackgroundColor
+        backgroundView = UIView()
+        backgroundView!.backgroundColor = SettingsUX.TableViewHeaderBackgroundColor
         addSubview(titleLabel)
         addSubview(topBorder)
         addSubview(bottomBorder)
+    }
 
+    override func layoutSubviews() {
         setupInitialConstraints()
```

Comment:
layoutSubviews can be called many times during the lifecycle of a view. Setting up constraints in layoutsubview is usually not a good idea. Is there a reason why we need to do this here?